src/routes/add-invoice.test.tsx

```diff
@@ -0,0 +1,121 @@
+    await user.type(nameField, 'Test Company');
+
+    const today = new Date();
+    const twoWeeksFromToday = addDays(today, 14);
+
+    const todayFormated = format(today, 'MM/dd/yyyy');
+    const twoWeeksFromTodayFormated = format(twoWeeksFromToday, 'MM/dd/yyyy');
+
+    const createdField = screen.getByLabelText('Created') as HTMLInputElement;
+
+    const validField = screen.getByLabelText('Valid until') as HTMLInputElement;
+
+    const recipientForm = screen.getByRole('group', { name: 'Recipient' });
+
+    const recipientCompanyNameField = within(recipientForm).getByLabelText(
+      'Company name',
+    ) as HTMLInputElement;
+    await user.type(recipientCompanyNameField, 'Test');
+
+    const recipientCityField = within(recipientForm).getByLabelText(
+      'City',
+    ) as HTMLInputElement;
+    await user.type(recipientCityField, 'Testopolis');
+
+    const recipientStreetField = within(recipientForm).getByLabelText(
+      'Street',
+    ) as HTMLInputElement;
+    await user.type(recipientStreetField, 'Test St.');
+
+    const recipientPostcodeField = within(recipientForm).getByLabelText(
+      'Postcode',
+    ) as HTMLInputElement;
+    await user.type(recipientPostcodeField, '12345');
+
+    const recipientNipField = within(recipientForm).getByLabelText(
+      'NIP',
+    ) as HTMLInputElement;
+    await user.type(recipientNipField, '1234567890');
+
+    const recipientPhoneField = within(recipientForm).getByLabelText(
+      'Phone',
+    ) as HTMLInputElement;
+    await user.type(recipientPhoneField, '1234567890');
+
+    const recipientEmailField = within(recipientForm).getByLabelText(
+      'Email',
+    ) as HTMLInputElement;
+    await user.type(recipientEmailField, 'john.doe@email.com');
+
+    const recipientBankAccountField = within(recipientForm).getByLabelText(
+      'Bank account',
+    ) as HTMLInputElement;
+    await user.type(recipientBankAccountField, 'PL27109024026154175215614473');
+
+    await user.click(screen.getByText('Add item'));
+
+    const orderItemForm = screen.getByRole('group', { name: 'Items' });
+
+    const orderItemNameField = within(orderItemForm).getByLabelText(
+      'Name',
+    ) as HTMLInputElement;
+    await user.type(orderItemNameField, 'Test item');
+
+    const orderItemAmountField = within(orderItemForm).getByLabelText(
+      'Amount',
+    ) as HTMLInputElement;
+    await user.type(orderItemAmountField, '1');
+
+    const orderItemUnitField = within(orderItemForm).getByLabelText(
+      'Unit',
+    ) as HTMLInputElement;
+    await user.type(orderItemUnitField, 'Test');
+
+    const orderItemTaxField = within(orderItemForm).getByLabelText(
+      'Tax',
+    ) as HTMLInputElement;
+    await user.type(orderItemTaxField, '23');
+
+    const orderItemPriceField = within(orderItemForm).getByLabelText(
+      'Price',
+    ) as HTMLInputElement;
+    await user.type(orderItemPriceField, '100');
+
+    expect(nameField.value).toBe('Test Company');
```

Comment:
I would say there is no need to test such behaviour. This is handled by the browser, so I'm pretty sure it is really well tested :D